locallib.php

```diff
@@ -582,8 +582,10 @@ function zoom_get_selectable_alternative_hosts_list(context $context) {
         // At least, Zoom does not care if the user who is the host adds himself as alternative host as well.
 
         // Verify that the user really has a Zoom account.
+        // Furthermore, verify that the user's status is active. Adding a pending or inactive user as alternative host will result
+        // in a Zoom API error otherwise.
         $zoomuser = $service->get_user($u->email);
-        if ($zoomuser !== false) {
+        if ($zoomuser !== false && $zoomuser->status == 'active') {
```

Comment:
As 'active' is known to be a string, a strict comparison here is precise, safe and efficient.
```suggestion
        if ($zoomuser !== false && $zoomuser->status === 'active') {
```

Reply:
Thanks for this note (again!). I keep forgetting about strict comparisons when working with PHP.